app/main.ipynb

```diff
@@ -124,10 +168,28 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "# write your code here"
+    "merged_df['order_hour'] = pd.to_datetime(merged_df['datetime']).dt.hour\n",
+    "order_hour_income = merged_df.groupby('order_hour')['price'].sum()\n",
+    "plt.figure(figsize=(8, 6))\n",
+    "plt.bar(order_hour_income.index, order_hour_income.values)\n",
+    "plt.xticks(np.arange(24))\n",
```

Comment:
You can pass the next attributes as arguments directly to the plt.bar(). Mind line length, don't forget split to improve readability in that case